src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/ChipClient.kt

```diff
@@ -43,7 +44,8 @@ object ChipClient {
     getAndroidChipPlatform(context)
 
     if (!this::chipDeviceController.isInitialized) {
-      chipDeviceController = ChipDeviceController()
+      /* 0xFFF4 is a test vendor ID, replace with your assigned ID */
```

Comment:
even though the comment explains things, generally we probably want to avoid magic numbers. Something like:

```java
static final int kTestVendorId = 0xFFF4; // 0xFFF1 to 0xFFF4 are test vendor IDs
```

likely makes the code more readable.